src/voodoo/spellbook/spells/trueStrike.ts

```diff
@@ -42,6 +42,7 @@ export const trueStrike: SpellFunction = async (voodoo, accountId, upgradeConfig
 
   for (const playerId of playerIds) {
     const baseDamage = await voodoo.getPlayerCheckStatBase({ accountId: playerId, stat: 'damage' });
+    const currentDamage = await voodoo.getPlayerCheckStatCurrent({ accountId: playerId, stat: 'damage' })
 
     if (baseDamage) {
```

Comment:
Implementing mdigena's suggestions
```suggestion
    const buffedDamage = baseDamage * (1 + bonus);
    
    const damageDelta = buffedDamage - currentDamage;
    
    
    if (damageDelta > 0) {
```